README.md

```diff
@@ -2,6 +2,52 @@
 
 A place to define the conventions we use to build APIs
 
+<!-- TOC depth:6 withLinks:1 updateOnSave:1 -->
+- [API styleguide](#api-styleguide)
+	- [Basic Tools](#basic-tools)
+	- [Versioning](#versioning)
+<!-- /TOC -->
+
 ## Basic Tools
 
 It's quite common in Hyper to build APIs with *ruby on rails*, so most of this conventions are designed to work well with Rails.
+
+## Versioning
+
+On standart rails applications, there are 3 used schemas for versioning
+
+
+  * Url
+
+  ```
+  GET hyper.super-app.no/api/v1.2/spaceships
+  ```
+
+  * Query parameters
+
+  ```
+  GET hyper.super-app.no/api/spaceships?version=1.2
+  ```
+
+  * Accept Header (Chosen approach)
+
+  ```
+  GET hyper.super-app.no/api/spaceships
+  ```
+
+  request headers
+
+  ```
+  Accept: application/vnd.super-app+json; version=1.2
```

Comment:
What do you think about these kind of versioning?
 GET hyper.super-app.no/api/v1/spaceships